css/styles.css

```diff
@@ -53,83 +53,96 @@ img {
   height: 20px;
 }
 
-h1 { 
-    text-align: center;
-    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+h2 {
+  text-align: center;
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  padding-top: 20px;
 }
```

Comment:
Added a bit of padding to `h2`. I changed your `h1` elements to `h2` in the HTML.